tests/lib_test.php

```diff
@@ -144,7 +144,7 @@ public function test_clean_envbars() {
         $data->showtext = 'Test Inject';
         envbarlib::update_envbar($data);
 
-        $injected = envbarlib::reinject();
+        $injected = envbarlib::reset_injectcalled();
 
         // An envbar is not in the database get_config('additionalhtmltopofbody').
         $additional = get_config('core', 'additionalhtmltopofbody');
```

Comment:
and here, I'd grep for additionalhtmltopofbody, it should not exist in the entire plugin at all after this patch

Reply:
Okay, I'll have a do this grep now

Reply:
A rgrep gives no results:
```
~local/envbar$ rgrep 'additionaltopofhtml'
~local/envbar$ 
```